kafka/resource_kafka_topic.go

```diff
@@ -50,23 +50,31 @@ func kafkaTopicResource() *schema.Resource {
 				Elem:        schema.TypeString,
 			},
 		},
+		Timeouts: &schema.ResourceTimeout{
+			Create: schema.DefaultTimeout(10 * time.Minute),
+			Read:   schema.DefaultTimeout(10 * time.Minute),
+			Update: schema.DefaultTimeout(10 * time.Minute),
+			Delete: schema.DefaultTimeout(10 * time.Minute),
+		},
```

Comment:
Standard terraform CRUD timeouts
https://www.terraform.io/plugin/sdkv2/resources/retries-and-customizable-timeouts